packages/opencensus-core/src/tags/validation.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {TagKey, TagValue} from './types';
+
+// String that has only printable characters
+const invalidString = /[^\u0020-\u007e]/;
```

Comment:
Nit: what would you think about calling this `nonPrintableCharsRegex`? If you did that could you remove the comment? I find the `String` in the comment and name a bit confusing.